modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.es.js

```diff
@@ -85,6 +85,10 @@ function toArray(value = '') {
 	}
 
 	if (!Array.isArray(newValue)) {
+		if (typeof newValue !== 'string') {
+			newValue = JSON.stringify(newValue);
+		}
+
```

Comment:
@marcelabc, can you please give me more details why it was necessary? Thanks.

Reply:
The predefined value of the Boolean field is a select field. The options are True/False, I put as a String (you can see the CheckboxDDMFormField*Settings). I don't know why this value is coming as boolean and not string in the frontend.

I chose to put this condition in this method because is it the place to transform/adapt the data to the expected one by the select field (array of string).

Reply:
@marcelabc, can we talk about this later today?

Thanks.

Reply:
I just talked to @marcelabc about this specific change and she'll analyze why this conversion is being necessary in the front-end. Maybe, it should be performed in the back-end.